announcements/1.16.4.md

```diff
@@ -0,0 +1,19 @@
+### Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.16.4.
+
+This is an out-of-order release to mitigate a bug found with newer versions of LXD, specifically the 5.0.2 update to the 5.0 LTS release series. The bug prevents containers from starting when LXD 5.0.2 is installed. If you're running LXD 5.0.2 you must update the Anbox image you're using.
+
+Please see the [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+#### New features & improvements
+
+n/a
+
+#### Bugs
+
+* AC-1320 Anbox containers fails to start with LXD 5.0.2
+
+## Upgrade instructions
+
+See [Upgrade Anbox Cloud](https://anbox-cloud.io/docs/howto/update/upgrade-anbox) or [Upgrade the Anbox Cloud Appliance](https://anbox-cloud.io/docs/howto/update/upgrade-appliance) for instructions on how to update your Anbox Cloud deployment to the 1.16.3 release.
```

Comment:
```suggestion
See [Upgrade Anbox Cloud](https://anbox-cloud.io/docs/howto/update/upgrade-anbox) or [Upgrade the Anbox Cloud Appliance](https://anbox-cloud.io/docs/howto/update/upgrade-appliance) for instructions on how to update your Anbox Cloud deployment to the 1.16.4 release.
```

Reply:
Is this actually true, since we don't provide an updated package?
Or do users just need to update the images (or actually just wait until the images update)?

Reply:
We have a part in the upgrade guide about `Upgrade LXD image` which applies here. They may need to do actual manual updates for apply the new images.

Reply:
But this section is only in the guide for updating the full deployment. Should it be added for the appliance?